docs/podman-remote.conf.5.md

```diff
@@ -0,0 +1,47 @@
+% podman-remote.conf(5)
+
+## NAME
+podman-remote.conf - configuration file for the podman remote client
+
+## DESCRIPTION
+The libpod.conf file is the default configuration file for all tools using
+libpod to manage containers.
+
+The podman-remote.conf file is the default configuration file for the podman
+remote client.  It is in the TOML format.  It is primarily used to keep track
+of the user's remote connections.
+
+## CONNECTION OPTIONS
```

Comment:
What if I want to give the whole bridge string?

Reply:
i assume you mean via env variable? then that overrules config file usage.

Reply:
You don't allow me to mix bridge strings and host ip's for different hosts in same config file.

Reply:
no